cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -603,10 +603,14 @@ You can also use:
 - [JohnTheRipper](https://github.com/hashcat/hashcat/issues/1057#issuecomment-279651700) to perform the password dictionary attack.
 - [Hashcat](https://hashcat.net/hashcat/).
 
+### Additional Reading
+
+- [JWT Best Practices Internet Draft](https://datatracker.ietf.org/doc/draft-ietf-oauth-jwt-bcp/)
```

Comment:
So far we are not accepting `Drafts` in general, as in, work in progress projects. @righettod what do you think about this one though?